### DISCUSSION TITLE
Hammerdb for sqlserver drops numbers by ~40% when the user has db_owner vs sysadmin

### DISCUSSION THREAD

**Discussion by sravandoma** [TPC-Council/HammerDB#127] [Q&A] [answered]
Hi,

We are seeing different performance numbers when the user has db_owner vs when the user is the owner of the database. This seems to be specific to hammerdb load. We are seeing ~35 - 40% performance drop.

Does this have to do with hammerdb? Is this expected? Looks like it is taking more time to process each wait event when the user has db_owner privs.

This is reproducable. I created 50 warehouses with 160 vusers.

Thanks.

**Comment by sm-shaw** [comment, marked as answer]
Hi,
Can you provide some more information on the steps you are doing to create the database and run the tests and any additional privilege settings you may be doing.  As an example on a test system if I create a test database called tpcc then under the database properties viewed with SSMS the owner is the account I logged in with using Windows authentication.  If you then browse the database all objects are explicitly prefixed by dbo on creation.  When you say specific to hammerdb load are you comparing this to other workloads that are also running at the same level of intensity and what sort of performance are you seeing?  Of course every database operation will check that the user has the prvilieges to do it however it should not be expected to have such a major performance overhead. If you can provide more details on how to reproduce what you are doing then it can be tested here as well.

**Comment by sm-shaw** [comment]
It has not been possible to reproduce this so far so further information is needed on the full configuration to investigate further.

**Comment by sm-shaw** [comment]
closed as no further follow up
